apps/gallery/js/startup.js

```diff
@@ -146,8 +146,9 @@ navigator.mozL10n.once(function startup() {
       }
 
       // Hide the scanning indicator
-      $('progress').classList.add('hidden');
+      // setTimeout() is to workaround Bug 1166500
       $('throbber').classList.remove('throb');
+      setTimeout(function() { $('progress').classList.add('hidden'); }, 100);
```

Comment:
Do we actually need a timeout of 100ms, or are you just trying to let the event loop fire in between these calls? If the latter, I'd omit the `100` entirely (`setTimeout` defaults to a timeout of 0ms).